jvm/src/test/scala/tastyquery/TypeSuite.scala

```diff
@@ -662,4 +662,9 @@ class TypeSuite extends UnrestrictedUnpicklingSuite {
     assert(clue(body.tpe).isOfClass(IntClass))
   }
 
+  testWithContext("BooleanRef") {
+    // val BooleanRef = resolve(name"scala" / name"runtime" / tname"BooleanRef").asClass
+    val scalaRuntime = resolve(name"scala" / name"runtime").asInstanceOf[DeclaringSymbol]
+    scalaRuntime.declarations
```

Comment:
Added this failing test to show that the reading of Java signatures is too eager.

Reply:
I also added the rt.jar in the `JavaTestPlatform` to run this test, but only when running in Java 8, which is why this test is not yet compatible with other Java versions.

To read Java 9+ class files, one could use something like this:
https://github.com/scalacenter/scala-debug-adapter/blob/0a25d3424bf5acfb4dcbebeada8fae980f4dc4bb/core/src/main/scala/ch/epfl/scala/debugadapter/ClassSystem.scala#L28-L36

Reply:
The problem is described in https://github.com/scalacenter/tasty-query/issues/80 and it should probably fixed in another PR.